twinn_ml_interface/objectmodels/logging.py

```diff
@@ -36,8 +36,10 @@ class MetaDataLogger:
     params: dict[str, str]
     artifacts: dict[str | PathLike, str | None]
     db_logs: dict[str, Hashable]
+    artifact_dir: PathLike | None
 
-    def __init__(self):
+    def __init__(self, artifact_dir: PathLike | None = None):
+        self.artifact_dir = artifact_dir
```

Comment:
self.artifact_dir should be cast as Path, that would solve some problems later on when using it like this: `self.artifact_dir / "somethingsomething"`